shared/reducers/entities.js

```diff
@@ -14,7 +14,15 @@ export default function(state: State = initialState, action: Actions): State {
     case 'entity:delete': {
       const {keyPath, ids} = action.payload
       // $FlowIssue doesn't understand this API
-      return state.updateIn(keyPath, map => map.deleteAll(ids))
+      // works in flow 4
+      // return state.updateIn(keyPath, map => map.deleteAll(ids))
```

Comment:
this does work in immutable 4 but not 3 which is on master now (coming soon)
and this is the only thing that uses this so far